### DISCUSSION TITLE
Week 7: July 29-Aug 2

### DISCUSSION THREAD

**Discussion by eeholmes** [SAFS-Varanasi-Internship/Summer-2024#21] [General]
Working on the AGU abstract.

Submit here:
https://agu.confex.com/agu/agu24/prelim.cgi/Session/225086

**Comment by eeholmes** [comment]
@yifeihang @ShridharS19 

Tuesday:

* I moved the abstract to Google Docs to make the joint editing easier with suggest mode since it is just text. https://docs.google.com/document/d/1klCanNy2FbtbGUZFECIx8havoqdJISJ-gbxEsexUFLo/edit?usp=sharing
* One of you will need to join AGU in order to submit the abstract. Cost is $29 for undergrads
* Then log in, and go here https://agu.confex.com/agu/agu24/prelim.cgi/Session/225086 to start the process of prepping the abstract.
* Add me as the 3rd author. Search for me via eli.holmes@noaa.gov and choose the 2nd one listed. That one should come up with the right affiliation for me.
* Yifei: Create a figure to include for the abstract. I think one of @yifeihang's  4 panel plots with a) original, b) prediction for original, c) prediction for full area, d) prediction for full area minus the gapfree Copernicus product. Is a good visual.

**Reply by yifeihang** [reply]
I created an account and joined the membership, but when I tried to login it keeps timing out. 
![image](https://github.com/user-attachments/assets/f1202f71-8153-4182-a0e4-2cff8df40e1b)

I added a 4 panel plot at the end of our google docs

**Reply by yifeihang** [reply]
Nevermind I think the abstract submission portal works

**Comment by eeholmes** [comment]
I moved the abstract to Google Docs to make the joint editing easier with suggest mode since it is just text. https://docs.google.com/document/d/1klCanNy2FbtbGUZFECIx8havoqdJISJ-gbxEsexUFLo/edit?usp=sharing

**Comment by eeholmes** [comment]
I will see you at 4pm PST.

**Comment by yifeihang** [comment]
[AGU_Submission.pdf](https://github.com/user-attachments/files/16447781/AGU_Submission.pdf)

**Comment by eeholmes** [comment]
Talk at CoastWatch. 20 minutes. This audience is familiar with ocean color.

1) Intro slide. What is the problem. a) lots of gaps. b) methods that use spatial interpolation, don't work well when there are lots of missing values.
2) Intro on methods. Why include environmental information (wind, sst, etc)?
3) What environment variables are most informative? Shridar's preliminary feature importance.
4) Explain U-Net
7) Explain PINN
8) Prediction evaluation methods
9) Results: comparing the estimates from different
10) Predicting using only environmental variables: method and results
11) Comparison to other gapfree data (Copernicus gapfree).

**Comment by eeholmes** [comment]
Falls under encoder-decoder models. Take image low-res preserving only important features (like ridge coming off africa, or blobs), then rebuild to high-res uses the training.

* image as an image as something real. looks for patterns.
* image as a special class of observations. objects are continuous in a weird way. 
* treat is as looking at objects

Why are CNNs a choice for gap-filling Chl-a?

* CHl-a is an image and it has features, like ridges coming off Africa, and we need the algorithm to learn those features and how they relate to the layers (variables) that we put into the model.

https://www.sciencedirect.com/science/article/pii/S1569843224003832

**Comment by eeholmes** [comment]
Friday:

link to abstract: https://docs.google.com/document/d/1klCanNy2FbtbGUZFECIx8havoqdJISJ-gbxEsexUFLo/edit

link to slides: https://docs.google.com/presentation/d/1etA0GIpuRJrahZnKkC36YrCN8A0dmnXPTETdv5v1m0E/edit#slide=id.p
